src/main/java/seedu/address/model/interview/predicate/ThisMonthWithinTimePeriodPredicate.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.interview.predicate;
+
+import java.time.LocalDateTime;
+import java.util.function.Predicate;
+
+import seedu.address.model.interview.Interview;
+
+/**
+ * Tests that a {@code Interview}'s date and time is within the next month.
+ */
+public class ThisMonthWithinTimePeriodPredicate extends WithinTimePeriodPredicate implements Predicate<Interview> {
+    private final LocalDateTime endDateTime;
```

Comment:
Not a functionality or "correctness" issue, just note that since the superclass already has the variable, there's not really a need to create a new one in the subclass!